test/fixtures/v5/turn/uturn_default.json

```diff
@@ -9,7 +9,7 @@
     },
     "instructions": {
         "ar": "انعطف نحو الدوران للخلف",
-        "da": "Foretag et U-vending",
+        "da": "Drej til U-vending",
```

Comment:
Should be "Foretag en U-vending"

Reply:
So "Foretag et" for straight and "Foretag en" for u-turn modifiers?

Reply:
No, "Foretag en" for U-turn. "Foretag" should not be used for straight modifiers. Do you see examples of that anywhere that I should fix?